server/compute.js

```diff
@@ -20,6 +22,46 @@ function applyUtils(sandbox = {}) {
   sandbox._ = deepFreeze(lodash);
 }
 
+const buildPayload = (pld, pl = {}) => {
```

Comment:
@KgaevskiT I'm not sure why we need to parse the `traits` arguments like this.
Couldn't we just pass the arguments directly to real `client.asUser().traits` or even expose the real method the the processor sandbox?